ci/benchmark/build.sh

```diff
@@ -37,7 +37,7 @@ export GBENCH_BENCHMARKS_DIR="$WORKSPACE/cpp/build/gbenchmarks/"
 export LIBCUDF_KERNEL_CACHE_PATH="$HOME/.jitify-cache"
 
 # Dask & Distributed git tag
-export DASK_DISTRIBUTED_GIT_TAG='2022.01.0'
+export DASK_DISTRIBUTED_GIT_TAG='main'
```

Comment:
I wonder if we should think about using the new nightly dask/distributed pakcages as opposed to the current pip/source installation?

Reply:
At times when our CI is blocked by a dask upstream change, merging a PR in dask upstream will unblock us almost instantly. Are the nightly packages published per PR or per day?

@charlesbluca @quasiben any idea on the frequency of nightly packages?

Reply:
This was one of the goals of having those nightly packages. Namely to use on CI.

They are built for every commit and only take a couple of minutes to build and upload.

Reply:
Made changes to include nightly installs.